src/inmanta/__init__.py

```diff
@@ -16,7 +16,7 @@
     Contact: code@inmanta.com
 """
 
-COMPILER_VERSION = "2022.5"
+COMPILER_VERSION = "2022.6"
```

Comment:
```suggestion
COMPILER_VERSION = "2022.5.1"
```
Only patches for the EOS branch.